### DISCUSSION TITLE
I give my school certificate but impossible to send it

### DISCUSSION THREAD

**Discussion by AzaleeX** [community/community#64962] [GitHub Education]
### Hi everyone,

Hi 👋

To provide a clearer summary of my current situation: I am unable to upload the correct image because my high school does not provide me with a school email so I have to use my phone's camera. Unfortunately, I can't retrieve the image using the camera directly because the camera doesn't want to return it. As a result, I contacted the support team over three weeks ago, but have yet to receive a response. Also, the automated bot for submitting images to verify its accuracy consistently throws the same error and turns out to be impossible to use.

While it's my school certificate, whatever should pass, so I ask that a real human person process my request.

**Comment by arafajs** [comment]
I would recommend that you try contacting the support team again. In your email, be sure to include the following information:

1. Your name and contact information
2. A brief description of the problem you are having
3. The steps you have already taken to try to resolve the problem
4. Any screenshots or error messages you have received

**Reply by AzaleeX** [reply]
I'm not their email
